src/Nethermind/Nethermind.JsonRpc.Connectors/Nethermind.JsonRpc.Clients.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What is the purpose of `Nethermind.JsonRpc.Clients` project?

Reply:
It was created to prevent a circular dependenciy, but since it was refactored it is no longer necessary.